utplsql-maven-plugin/src/main/java/org/utplsql/helper/ReporterDefault.java

```diff
@@ -0,0 +1,31 @@
+package org.utplsql.helper;
+
+import org.utplsql.api.reporter.Reporter;
+
+/**
+ * This class is an enumeration of all the known reporter in {@code utPLSQL}. It further more defines the default output
+ * file for each {@link Reporter}. In case the output file is set to {@code -}, it will mean the stdout of the process.
+ * 
+ * @author Alberto Hernández
+ *
+ */
+public enum ReporterDefault {

```

Comment:
We should think about a different approach here in future. We plan to support custom reporters soon and therefore shouldn't rely on static lists of reporters

Reply:
One of our main problems with the plugin was the understanding the input parameters of the utplsql so we try to enforce and document the differents parameters furthermore maven usually try to offer an standard configuration for the user (convention over configuration).
We understand that is a valid approach that once you have implement new reporter in the utpsql package a new plugin release could be developed to implement this functionality.

Reply:
Every client/user can create their own reporters (outside of main utPLSQL project).
Even though it's not yet documented that was that main vision and idea behind it.
Once we cleanup reporters structure in utPLSQL project and this gets documented, users can have any reporter they want developed locally (in-house) for their own needs.

I would define reporters similarly to what we have in utPLSQL-cli parameters.
Example:
```xml
<reporters> <!--default - <reporter><name>ut_documentation_reporter</name></reporter>-->
  <reporter>
    <name>ut_xunit_reporter</name>
    <output_file>test_results.xml</output_file> <!--no default-->
    <output_to_screen>true</output_to_screen> <!--default - false-->
  </reporter>
</reporters>
```
In general, you could have local reproters (not part of utPLSQL project)
- `ut_save_to_table_reporter` reporter, that doesnt give any output, but save test results to table.
- `ut_minimal_reporter` - that just gives 'AWESOME!!!' as an output, when no tests failed.
- etc.

There are just few rules to reporter creation
- reporter constructor needs to have zero mandatory parameters.
- reporter needs to be defined under `ut_reproter_base`

There will be more coming, once I finalize cleanup and refactoring, but the refactoring will not impact consumers too much.